src/Config.hs

```diff
@@ -19,7 +19,7 @@ makeLenses ''Config
 
 config = Config
     { _dbPath = "./aula.db"
-    , _listenerInterface = "127.0.0.1"
+    , _listenerInterface = "0.0.0.0"
```

Comment:
Running aula-server from docker needs to listen on `0.0.0.0`. In my opinion we should use config files for even this small config options.